test.ts

```diff
@@ -14,24 +14,24 @@ export const test = base.extend<TestOptions>({
   user: [
     {
       name: 'Approved Premises E2ETester',
-      username: process.env.HMPPS_AUTH_USERNAME as string,
-      password: process.env.HMPPS_AUTH_PASSWORD as string,
+      username: (process.env.CAS2_HMPPS_AUTH_USERNAME as string) || (process.env.HMPPS_AUTH_USERNAME as string),
```

Comment:
Maybe I should write this as?

```
(process.env.CAS2_HMPPS_AUTH_USERNAME || process.env.HMPPS_AUTH_USERNAME) as string
```

Reply:
If we changed github actions to use the CAS2 naming style, would we still need to do this? It would break any other PR's checks temporarily, but only until this PR was merged?

Reply:
Yeah I still think we need to do this rename because this repo is consumed and used in two other contexts:

- [frontend](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/blob/bd9c1cd3e7a37c20eaa10403ba44dbef70764292/.circleci/config.yml#L168) 
- [api (new)](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1346)

We could change the github actions for this repo but I don't think that helps us run the e2e tests in those other contexts.

Reply:
Removed the conditional https://mojdt.slack.com/archives/C05GEPLBRN3/p1705510501914629?thread_ts=1705505858.383989&cid=C05GEPLBRN3